cmd/thriftrw-list-deps/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"go.uber.org/thriftrw/compile"
+)
+
+type args struct {
+	ThriftFile string `positional-arg-name:"file" description:"Path to the Thrift file"`
+}
+
+var opts struct {
+	RelativeTo string `long:"relative-to" description:"If specified, output paths will be relative to this directory"`
+	Args       args   `positional-args:"yes" required:"yes"`
+}
+
+// listDependentThrifts lists the Thrift files that the given Thrift file depends on or includes, both directly (through imports) and
+// indirectly (through transitive imports).
+//
+// The returned file paths are absolute, unless relativeDir parameter is given, in which case the paths are relative to
+// the relativeDir.
+func listDependentThrifts(inputThriftFile string, relativeDir string) ([]string, error) {
+	dependentThriftFiles := make([]string, 0)
```

Comment:
nit: use `var` form for empty slices.

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
Also, nit: Again, verbose variables not needed. Can use just `deps`.